taskvine/src/worker/vine_worker.c

```diff
@@ -1441,49 +1465,76 @@ static int enforce_worker_promises(struct link *manager)
 }
 
 /*
-Given a freshly started process, wait for it to initialize and send
-back the library startup message with JSON containing the name of
-the library, which should match the task's provides_library label.
+Kill a failed library process.
+This library process may either not existent or no longer responses.
 */
 
-static int check_library_startup(struct vine_process *p)
+static void handle_failed_library_process(struct vine_process *p, struct link *manager)
```

Comment:
I would like to minimize the number of ways in which libraries are "different" from ordinary tasks.
I think it's better if a library is reaped in exactly the same way as a normal process.

(And the functions left behind will still be forsaken quickly when they discover `process_can_run_eventually`
 is false.